src/getgbeh.f

```diff
@@ -1,167 +1,167 @@
-C>     @file
-C>     @brief Find a grib message.
+C> @file
+C> @brief Find a grib message.
+C> @author Mark Iredell @date 1995-10-31
+
+C>Read a grib index file (or optionally the grib file itself) to get
```

Comment:
Put a space between the C> and the start of text. So:

`C> Read a grib index file (or optionally the grib file itself) to get
`